### PR TITLE
Estonian 6.14.0

### PR DESCRIPTION
**Pull request by kristoheero**
* XTE-334 / Joint development issue #135: Security Server bugfix: Fixed not correctly functioning timeouts.
* XTE-337 / Joint development issue #140: Remove X-Road version 5 migration support. 
  **Warning:** Central Server database schema changed, old Central Server backups are not usable.
* XTE-341 / Joint development issue #142: Security Server: Enable detect connections that have become stale (half-closed) while kept inactive in the connection pool.

### CODE REVIEW

**Review by VitaliStupin — APPROVED** [review]

File: src/addons/messagelog/src/main/java/ee/ria/xroad/proxy/messagelog/TimestamperUtil.java [hunk 1]
```diff
@@ -50,48 +52,46 @@ private TimestamperUtil() {
...
-    static TimeStampToken addSignerCertificate(
-            TimeStampResponse tsResponse, X509Certificate signerCertificate)
-                    throws Exception {
+    static TimeStampToken addSignerCertificate(TimeStampResponse tsResponse,
+            X509Certificate signerCertificate) throws Exception {
         CMSSignedData cms = tsResponse.getTimeStampToken().toCMSSignedData();
 
-        List<X509Certificate> collection = Arrays.asList(signerCertificate);
+        List<X509Certificate> collection = Collections.singletonList(signerCertificate);
         collection.addAll(cms.getCertificates().getMatches(null));
```
Comment (jhyoty) [inline src/addons/messagelog/src/main/java/ee/ria/xroad/proxy/messagelog/TimestamperUtil.java]: The _collection_ is immutable (a singletonlist). AddAll will fail if the argument is not an empty collection.

Comment (kristoheero) [reply src/addons/messagelog/src/main/java/ee/ria/xroad/proxy/messagelog/TimestamperUtil.java]: Thanks, mistake fixed.

Comment (jhyoty) [reply src/addons/messagelog/src/main/java/ee/ria/xroad/proxy/messagelog/TimestamperUtil.java]: Also the previous version contains a bug: Arrays.asList returns a fixed-size list, so addAll can fail.

Comment (kristoheero) [reply src/addons/messagelog/src/main/java/ee/ria/xroad/proxy/messagelog/TimestamperUtil.java]: True, true..  :+1:

File: src/packages/xroad-jetty9/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+xroad-jetty9 (6.14.0) trusty; urgency=low
+
```
Comment (jhyoty) [inline src/packages/xroad-jetty9/debian/changelog]: Version should be 6.14.0-1

File: src/packages/xroad/debian/changelog [hunk 3]
```diff
@@ -1,3 +1,16 @@
+xroad (6.14.0) trusty; urgency=low
+
```
Comment (jhyoty) [inline src/packages/xroad/debian/changelog]: Version should be 6.14.0-1

Comment (kristoheero) [reply src/packages/xroad/debian/changelog]: This is something new for us. So fare, we are always used version pattern x.y.z? Where is this new requirement presented?

Comment (jhyoty) [reply src/packages/xroad/debian/changelog]: The software version is still 6.14.0. The "1" part is the packaging version, which is a .deb (and rpm) convention we have been following. A .deb package version number consist of the upstream software version (here 6.14.0) and the packaging version (here 1), separated by a hyphen.

Comment (kristoheero) [reply src/packages/xroad/debian/changelog]: Done.

**Review by jhyoty — APPROVED** [review]

**Review by iluwatar — APPROVED** [review]

**Pull request merged**